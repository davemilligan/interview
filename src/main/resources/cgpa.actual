LtbWRlEpzNbawIIo
ssLafrVwU
gmWXph
uMwoJFugvXGnSgaApSIzNN
VSgQTgiqNyQMRerhQdLRUemcEQX
ibyUYmXdfoAW
UhhdgCqcOuWAJOFWKqsGUGw
ABAJwYeibIJQ
hlTgCgoFvpFUSXTssbQ
bsbxNbUVvECrfSV
AVSHnKvDZorlL
IwQmGhDJAiExVBjUmHXAfvUDqUyksh
XvxXsDKIjkFJHiWTOkd
aOdxZenFiSfZzoO
yvOgyJLb
bGVtEfpjOniybCxKYVrPMkqbhVYsG
VDhZY
sydIngROEZHupsHzPFFpXhZh
JXneisxgkeLkak
VkNpJJXfFnzBclOPwIvdabTxqSzFJ
DHRSaJQJdJiJBmxjmLouQ
mdOtOeFpyLW
mcNnINTMSdpMxwTApWSJ
EcKbvrkcWEkqZZMHXnqzjTSG
pVGMylZEC
PDpUZLlXiihnpYCmDCor
KUlxpQpgbCPp
KkKNPuDXiddAsnCQZgnMQyhJqB
vZuGpylfeQrliAXfJT
FfMtpOUNvGKBETc
hyDbVOGaobNkXZR
jTkQkHxwAKfyykjWiyJKTqcCUZ
EnSVRSXzsyWuCMVVe
ZZDGEToBqnxnPK
qJOdMwks
XhhoNTJ
wCZQpyjUBfEmZpApljyxKbPv
HYasEeNIkTsDSnuMXQGDuFIZuGen
euKDKBZrwqODRtIyWji
vosdf
QumKkEcRFIdHaTHudcvHjEAYlAnkn
hYGUutHgTzGQhWM
QryjiTfrkcUGnJBnkzAWwdiqJhZqh
bHfivIcusSyRnaovkBZazI
PdeLwCYcXHBUyDovShzHKI
fwXRzErDWcyKjPxbOfApVJx
agPfUmvm
nTBfkiqGrAem
RzXoSUoHEcIH
dwBNMwtQwYeZprqQpXK
LdHAmtvQLIEpXoWwmhfEDmguyJ
NnDGDcEQ
QWPwPuN
WneXKKERUBQ
XgBjLqwusge
OllfZlACfxPdXgsM
FxwndjRHEXgXaXYw
ZlusqchpsvpSimDBnocPXdgCW
dofSxfFEwPPnKwFfVsynJkBDhumIA
NElkBCFZUUJBLLcVjQn
PrvDm
tVsahPGsuJFIQ
ArFnJ
PnfTXKEvdJeLaciXAIooDmqlTuBxJL
LIaCQ
SjqGgkIlXdWxmVgZkrqgaBX
jadiayxBP
tMloJlO
FEXmMZqytnoWRMACuad
hjBCAXDgihUqPExDuRrJcz
ggjwTvvaAraioUNmKKfKWwA
mzEhSqlRtxslyNZZnmCGGdvJbujoIe
CEoQmJhGNuufStMmSAHCj
BmnINxoKI
vcGdlaKlNsEp
tzqGKOBIHnl
FSXEeOwelNxMVutJDFjEzLm
xvIUeYZMlBrRbAUxcLtzLPdjORQIYZ
yNXtoxp
wAEauaPeHuTdnyQY
YVTVYxRUiAoFNNgHJGyF
dkrlgltSHdWFTvldivCmKGIRdp
oWgzOedgOdRozRaEymNOLPRCqvl
fmbSzWBAFYvgvYoAfNgh
NQfoFodr
MkVKGyExPayZExpJEl
WJTKFI
toTIoHWsZB
tNDpFOSdpdUfUbzkcINKdHIaZxV
FlbyiCXNChMaokOglrGd
xcRgist
NDnPzRMahqJNvqArVFrqrZOphOAHO
XHjqMumVZfSW
zXiGscYethmwLgalEVwuSGdzxI
fRHKrHh
YAmyNFyT
rAFMZelAhtAYs
KvzKzel
keyYAGA
FOIzcUHOGYTBnGLn
