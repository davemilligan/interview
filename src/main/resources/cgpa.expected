LtbWRlEpzNbawIIo
ssLafrVwU
gmWXph
uMwoJFugvXGnSgaApSIzNN
VSgQTgiqNyQMRerhQdLRUemcEQX
ibyUYmXdfoAW
UhhdgCqcOuWAJOFWKqsGUGw
ABAJwYeibIJQ
hlTgCgoFvpFUSXTssbQ
bsbxNbUVvECrfSV
AVSHnKvDZorlL
IwQmGhDJAiExVBjUmHXAfvUDqUyksh
XvxXsDKIjkFJHiWTOkd
aOdxZenFiSfZzoO
yvOgyJLb
bGVtEfpjOniybCxKYVrPMkqbhVYsG
VDhZY
sydIngROEZHupsHzPFFpXhZh
JXneisxgkeLkak
VkNpJJXfFnzBclOPwIvdabTxqSzFJ
DHRSaJQJdJiJBmxjmLouQ
mdOtOeFpyLW
mcNnINTMSdpMxwTApWSJ
EcKbvrkcWEkqZZMHXnqzjTSG
pVGMylZEC
PDpUZLlXiihnpYCmDCor
KUlxpQpgbCPp
KkKNPuDXiddAsnCQZgnMQyhJqB
vZuGpylfeQrliAXfJT
hyDbVOGaobNkXZR
FfMtpOUNvGKBETc
jTkQkHxwAKfyykjWiyJKTqcCUZ
EnSVRSXzsyWuCMVVe
ZZDGEToBqnxnPK
qJOdMwks
XhhoNTJ
wCZQpyjUBfEmZpApljyxKbPv
HYasEeNIkTsDSnuMXQGDuFIZuGen
euKDKBZrwqODRtIyWji
vosdf
QumKkEcRFIdHaTHudcvHjEAYlAnkn
hYGUutHgTzGQhWM
QryjiTfrkcUGnJBnkzAWwdiqJhZqh
bHfivIcusSyRnaovkBZazI
PdeLwCYcXHBUyDovShzHKI
fwXRzErDWcyKjPxbOfApVJx
agPfUmvm
nTBfkiqGrAem
RzXoSUoHEcIH
dwBNMwtQwYeZprqQpXK
LdHAmtvQLIEpXoWwmhfEDmguyJ
NnDGDcEQ
QWPwPuN
WneXKKERUBQ
XgBjLqwusge
OllfZlACfxPdXgsM
FxwndjRHEXgXaXYw
dofSxfFEwPPnKwFfVsynJkBDhumIA
ZlusqchpsvpSimDBnocPXdgCW
NElkBCFZUUJBLLcVjQn
PrvDm
tVsahPGsuJFIQ
ArFnJ
PnfTXKEvdJeLaciXAIooDmqlTuBxJL
LIaCQ
SjqGgkIlXdWxmVgZkrqgaBX
tMloJlO
jadiayxBP
hjBCAXDgihUqPExDuRrJcz
FEXmMZqytnoWRMACuad
ggjwTvvaAraioUNmKKfKWwA
mzEhSqlRtxslyNZZnmCGGdvJbujoIe
CEoQmJhGNuufStMmSAHCj
BmnINxoKI
vcGdlaKlNsEp
tzqGKOBIHnl
FSXEeOwelNxMVutJDFjEzLm
xvIUeYZMlBrRbAUxcLtzLPdjORQIYZ
yNXtoxp
wAEauaPeHuTdnyQY
YVTVYxRUiAoFNNgHJGyF
dkrlgltSHdWFTvldivCmKGIRdp
oWgzOedgOdRozRaEymNOLPRCqvl
fmbSzWBAFYvgvYoAfNgh
NQfoFodr
MkVKGyExPayZExpJEl
toTIoHWsZB
WJTKFI
tNDpFOSdpdUfUbzkcINKdHIaZxV
FlbyiCXNChMaokOglrGd
xcRgist
NDnPzRMahqJNvqArVFrqrZOphOAHO
XHjqMumVZfSW
zXiGscYethmwLgalEVwuSGdzxI
fRHKrHh
YAmyNFyT
rAFMZelAhtAYs
KvzKzel
keyYAGA
FOIzcUHOGYTBnGLn
